lib/shopify_theme/api_checker.rb

```diff
@@ -0,0 +1,39 @@
+module ShopifyTheme
+  class APIChecker
+    class APIResult
```

Comment:
I don't really see the point of it at the moment. So I figured I could just let it live in here.

If it starts to get more complicated then I'll probably move it out.